src/components/atoms/CircleProgress/CircleProgress.tsx

```diff
@@ -167,12 +173,16 @@ export default function CircleProgress({
           cy={squareSize / 2}
           r={initData.radius}
           strokeWidth={`${strokeWidth}px`}
+          style={progressData.circleBackgroundStyles}
+          transform={`rotate(${initData.arcRotation} ${squareSize / 2} ${
```

Comment:
the value of transform is exactly same for both `circle` elements, store in a var and use that instead.